src/services/formatting/formatting.ts

```diff
@@ -1117,6 +1117,27 @@ namespace ts.formatting {
         }
     }
 
+    /**
+     * @returns -1 iff the position is not in a multi-line comment.

```

Comment:
It's common to simply say "a negative value" rather than "-1" to retain flexibility for future refactoring.

Also, what does it return if it *is* in a multi-line comment?